differ/mutationDiffer.go

```diff
@@ -90,20 +90,39 @@ type MutationDiffer struct {
 	utils           xdcrUtils.UtilsIface
 }
 
+type JsonBody struct {
+	Value    []byte
+	Cas      uint64
+	Datatype uint8
+}
+
+type JsonMeta struct {
```

Comment:
JsonMeta -> JsonMetaWithBody

Make a comment saying that body could be optional